client/webserver/site/src/js/markets.ts

```diff
@@ -598,22 +599,36 @@ export default class MarketsPage extends BasePage {
 
   /* setHighLow calculates the high and low rates over the last 24 hours. */
   setHighLow () {
-    const cache = this.market?.candleCaches[fiveMinBinKey]
-    if (!cache) {
-      for (const s of this.stats) {
-        s.tmpl.high.textContent = '-'
-        s.tmpl.low.textContent = '-'
+    let [high, low] = [0, 0]
+    const spot = this.market.cfg.spot
+    // Use spot values for 24 hours high and low rates if it is available. We
+    // will default to setting it from candles if it's not.
+    if (spot && spot.low24 && spot.high24) {
+      high = spot.high24
+      low = spot.low24
+    } else {
+      const cache = this.market?.candleCaches[fiveMinBinKey]
+      if (!cache) {
+        if (this.candleDur !== fiveMinBinKey) {
+          this.candleDur = fiveMinBinKey
```

Comment:
I realized that doing this without doing `loadCandles` will cause some UI problems. We don't want to change this anyway. We want to show the user what they last selected. The changes [here](https://github.com/buck54321/dcrdex/compare/1fc4e44...buck54321:dcrdex:candles-dur-arg) allow the five-min candles to be loaded in the background and the UI to still function properly with the user's selection.